core/migration/phase.go

```diff
@@ -39,6 +39,19 @@ var phaseNames = []string{
 	"ABORTDONE",
 }
 
+// Those phases are only used to get a complete successful round for testing purposes.
+func GetCompletePhaseMigrationRound() []Phase {
```

Comment:
I don't like adding an exported function that is only used in tests, but I can't think of a suitable place to put it at this point.

In any case:
```go
// SuccessfulMigrationPhases returns the phases that a successful migration
// transitions through in correct order. It is used for testing.
func SuccessfulMigrationPhases() []Phase {
```

Reply:
I agree with this! I did not have any other place in mind though, for me, refactoring was still a win.